### PR TITLE
docs: Updates for 0.13.1.1 (testnet)

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

Updates for 0.13.1.1 (testnet)

### PR Preview URL

[Release notes](https://starknet-io.github.io/starknet-docs/pr-1208/documentation/starknet_versions/version_notes/)
[Gas and transaction fees > L2 payloads: calldata, events, and code](https://starknet-io.github.io/starknet-docs/pr-1208/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/#l2_calldata)
[Current limits](https://starknet-io.github.io/starknet-docs/pr-1208/documentation/tools/limits_and_triggers/)

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1208)

### CODE REVIEW

File: components/Starknet/modules/starknet_versions/pages/version_notes.adoc [hunk 1]
```diff
@@ -16,8 +16,26 @@ include::ROOT:partial$snippet_important_goerli2_removed.adoc[]
...
 
 |Mainnet|0.13.1|1.5.0|2.0.0 - 2.6
-|Sepolia Testnet|0.13.1|1.5.0|2.0.0 - 2.6
-|Goerli Testnet (deprecated)|0.13.1|1.5.0|2.0.0 - 2.6
+|Sepolia Testnet|0.13.1.1|1.5.0|2.0.0 - 2.6
+|===
+
+[id="version0.13.1.1"]
+== Starknet v0.13.1.1
+
```
Comment (ArielElp) [inline components/Starknet/modules/starknet_versions/pages/version_notes.adoc:24]: I think that we should add a sentence in the spirit of the mailing list update - following community feedback regarding high declaration prices ...

**Review by stoobie — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1208)* status: 0 of 4 files reviewed, 1 unresolved discussion (waiting on @ArielElp and @iliav-starkware)

___
*[`components/Starknet/modules/starknet_versions/pages/version_notes.adoc` line 24 at r1](https://reviewable.io/reviews/starknet-io/starknet-docs/1208#gh-1546535735:-NuSZzGE9lDQpJhayYbO:bfqbqea) ([raw file](https://github.com/starknet-io/starknet-docs/blob/4bd89275c9baa34d1fd652fea5a78397ce8209b9/components/Starknet/modules/starknet_versions/pages/version_notes.adoc#L24)):*
<details><summary><i>Previously, ArielElp wrote…</i></summary><blockquote>

I think that we should add a sentence in the spirit of the mailing list update - following community feedback regarding high declaration prices ... 
</blockquote></details>

@iliav-starkware Do you concur?

File: components/Starknet/modules/starknet_versions/pages/version_notes.adoc (hunk 1 above)
Comment (stoobie) [reply components/Starknet/modules/starknet_versions/pages/version_notes.adoc:24]: @ArielElp PTAL.

**Review by ArielElp — DISMISSED** [review]
:lgtm:

> Reviewed 3 of 4 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1208#-:-NuSbB3q6g05J1lb_6Ow:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @iliav-starkware)

**Review by iliav-starkware — APPROVED** [review]

**Pull request merged**